spec/models/validations/sales/sale_information_validations_spec.rb

```diff
@@ -467,4 +467,293 @@
       end
     end
   end
+
+  describe "#validate_stairbought" do
+    let(:now) { Time.zone.local(2024, 4, 4) }
+
+    before do
+      Timecop.freeze(now)
+      Singleton.__init__(FormHandler)
+    end
+
+    after do
+      Timecop.return
+      Singleton.__init__(FormHandler)
+    end
+
+    context "when ownership type is Shared Ownership (new model lease)" do
```

Comment:
Can these tests be parameterised at all?

Reply:
For sure!